### DISCUSSION TITLE
Different pricing for Enterprise if existing customer vs new?

### DISCUSSION THREAD

**Discussion by sjustason** [community/community#130789] [Enterprise] [answered]
In our existing Enterprise organization account it shows that every license is $39 USD / month, but looking at the pricing page it shows $21 USD / month, does anyone know why the different pricing scheme? https://github.com/pricing; Of course, I'm also curious why we can't have access to the team pricing at $4 USD / month for our organization, but that doesn't even seem to be on the table... I searched for similar questions, but unsurprisingly they went unanswered.

**Comment by queenofcorgis** [comment, marked as answer]
@sjustason - thanks for posting. I recommend opening up a ticket on our [Support page](https://support.github.com/contact) so our team can take a look at your account specifically and answer your questions in detail. 

You are likely unable to access a Team subscription given that you are already subscribed to Enterprise, Enterprise accounts are only available with GitHub Enterprise, so it's not possible to downgrade an enterprise account to another plan. Check out [this doc](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-the-plan-for-your-github-account/downgrading-your-accounts-plan#downgrading-your-enterprise-accounts-plan) for more information.

**Comment by sjustason** [comment]
Just a follow-up to this, I found through their support they can downgrade (manually, on their backend) from the $39 Enterprise pricing to the $19 Business pricing.  Setup was a bit annoying as you need to make sure you set "policies" on ALL options in the Enterprise Copilot policies page and the organizations Copilot policies page before you can assign seats.
